packages/app/src/cli/utilities/developer-platform-client.ts

```diff
@@ -0,0 +1,19 @@
+import {PartnersClient} from './developer-platform-client/partners-client.js'
+import {PartnersSession} from '../../cli/services/context/partner-account-info.js'
+import {MinimalOrganizationApp, Organization, OrganizationApp} from '../models/organization.js'
+
+export type Paginateable<T> = T & {
+  hasMorePages: boolean
+}
+
+export function selectDeveloperPlatformClient(): DeveloperPlatformClient {
+  return new PartnersClient()
+}
+
+export interface DeveloperPlatformClient {
+  session: () => Promise<PartnersSession>
```

Comment:
Do we want to define a new interface for this? `PartnersSession` is tied to the `PartnersClient` no?

Reply:
You're correct that this is undesirable. We have left this here for now as an escape route for existing paths, so we can gradually migrate over pieces of the system. Ultimately there should be no session on the public interface.

Reply:
That makes sense 👌